sdk/go/azure/network/virtualHub.go

```diff
@@ -64,24 +64,20 @@ type VirtualHub struct {
 	ResourceGroupName pulumi.StringOutput `pulumi:"resourceGroupName"`
 	// One or more `route` blocks as defined below.
 	Routes VirtualHubRouteArrayOutput `pulumi:"routes"`
+	// The sku of the Virtual Hub. Possible values are `Basic` and `Standard`. Changing this forces a new resource to be created.
+	Sku pulumi.StringPtrOutput `pulumi:"sku"`
 	// A mapping of tags to assign to the Virtual Hub.
 	Tags pulumi.StringMapOutput `pulumi:"tags"`
 	// The ID of a Virtual WAN within which the Virtual Hub should be created. Changing this forces a new resource to be created.
-	VirtualWanId pulumi.StringOutput `pulumi:"virtualWanId"`
+	VirtualWanId pulumi.StringPtrOutput `pulumi:"virtualWanId"`
```

Comment:
Looks like these String -> StringPtr changes might be breaking.

Reply:
There's nothing specific we can do when something has been made optional...

Reply:
I believe these are the kinda "breaking" changes we need to just accept